pointsource_imaging/MIRI_Aperture_Photometry_solution.ipynb

```diff
@@ -161,6 +161,18 @@
     "### 2.2<font color='white'>-</font>Plot configuration<a class=\"anchor\" id=\"plot\"></a> ###"
```

Comment:
<p>For this <code>pprint</code>, I would add  a <code>max_width</code> argument between 100-200 so the output is easier to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>